window-icon-updater/icon-sender

```diff
@@ -49,6 +52,7 @@ class IconRetriever(object):
         self.conn = xcffib.connect()
         self.setup = self.conn.get_setup()
         self.root = self.setup.roots[0].root
+        self.display = Display()
```

Comment:
Don't make the second X11 connection with another library, use XCB connection that is here already.